artifact/content/README.md

```diff
@@ -1,69 +1,119 @@
+
+
+
 ## Build Artifact
-The artifact image has been built with the Dockerfile included in the artifact.
-It is located at `/home/gobra/Dockerfile` or can be found at the root level in the shared folder when following the instructions given in [Running Gobra](#running-gobra).
-The same Dockerfile can also be found in [our repository](https://github.com/viperproject/gobra/tree/master/artifact).
-Because additional files are included in the artifact image (e.g. this README, the paper or some shell scripts), the following command should be executed in the `artifact` folder in our repository:
+
+The artifact image has been built with the Dockerfile included in this artifact.
+It is located at `/home/gobra/Dockerfile` or can be found in the shared folder on the host when following the instructions given in section ["Running Gobra"](#running-gobra). The same Dockerfile can also be found in [our repository](https://github.com/viperproject/gobra/tree/artifact1/artifact).
+Because additional files are included in the artifact image (e.g. this README, the paper, and some shell scripts), the following command has to be executed in the `artifact` folder in our repository:
 ```commandline
 docker build -t gobraverifier/gobra-artifact:v1 .
 ```
-Note that building the artifact image and executing the instructions given in the first section are performed and tested as part of our continuous integration to ensure that Gobra remains reusable by the community in the future.
-The CI status can be checked [here](https://github.com/viperproject/gobra/actions?query=workflow%3Aartifact+branch%3Amaster).
+Building the artifact image and executing the instructions given in the first section are performed and tested as part of our continuous integration to ensure that Gobra remains reusable by the community in the future.
+
+Our GitHub repository also has instructions on how to install Gobra from scratch without using Docker. These instructions can be found [here](https://github.com/viperproject/gobra/blob/artifact1/README.md).
+
+
+
+## License
 
 Gobra and its sources are licensed under the [Mozilla Public License Version 2.0](http://www.mozilla.org/MPL/2.0/) with the following exceptions:
 - The `.gobra` files used for regression testing are in [Public Domain](http://creativecommons.org/publicdomain/zero/1.0/)
-- Certain files containing stubs for Go built-ins or the Go standard library (located in `src/main/resources/builtin` and `src/main/resources/stubs/sync`, respectively) are copyrighted by The Go Authors because only minor changes have been performed to their counterparts in the Go source code.
+- Certain files containing stubs for Go built-ins or the Go standard library (located in `src/main/resources/builtin` and `src/main/resources/stubs/sync`, respectively) are copyrighted by the Go Authors ([license](https://golang.org/LICENSE)) because these files are largely inspired by their counterparts in the Go source code.
```

Comment:
```suggestion
- Certain files containing stubs for Go built-ins or the Go standard library (located in `/home/gobra/gobra/src/main/resources/builtin` and `/home/gobra/gobra/src/main/resources/stubs/sync`, respectively) are copyrighted by the Go Authors ([license](https://golang.org/LICENSE)) because these files are largely inspired by their counterparts in the Go source code.
```